designer/server/src/views/account/signed-out.njk

```diff
@@ -1,9 +1,10 @@
 {% extends 'layouts/page.njk' %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
```

Comment:
```suggestion
```

Reply:
Turns out my "better" Nunjucks extension isn't better, it doesn't flag this! Thanks.

Reply:
Done